sdks/java/io/parquet/src/main/java/org/apache/beam/sdk/io/parquet/ParquetIO.java

```diff
@@ -44,13 +44,13 @@
 import org.apache.beam.sdk.coders.CoderRegistry;
 import org.apache.beam.sdk.coders.StringUtf8Coder;
 import org.apache.beam.sdk.io.FileIO;
-import org.apache.beam.sdk.io.FileIO.ReadableFile;
```

Comment:
Can we get this one back again and and the shorter version in its use `ReadableFile` insted of `FileIO.ReadableFile` in the code/comments.

Reply:
Done. in [PR/13646](https://github.com/apache/beam/pull/13646)